jenkins/Dockerfile-blossom.integration.centos7

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+###
+#
+# Arguments: CUDA_VER=10.1 or 10.2
+#
+###
+ARG CUDA_VER=10.1
+
+FROM nvidia/cuda:${CUDA_VER}-runtime-centos7
+
+ARG CUDA_VER=10.1
```

Comment:
This arg appears twice?  Also wondering why these args don't appear up above with the block comment for args.

Reply:
good question. This was just copied from [existing Dockerfile.integration.centos7](https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/jenkins/Dockerfile.integration.centos7) w/ a few modification. Let me make them decent